buildSrc/src/main/groovy/com/jakewharton/kotlingen/KotlinGenPlugin.groovy

```diff
@@ -0,0 +1,37 @@
+package com.jakewharton.kotlingen
+
+import com.android.build.gradle.api.LibraryVariant
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+/**
+ * Plugin for generating kotlin bindings for Rx*.java implementations.
+ *
+ * For every project that applies this plugin, a corresponding "generateKotlinFor<project name>" task
+ * is created that you can run.
+ *
+ * Example: applying this in the rxbinding module results in a task called "generateKotlinForRxbinding"
+ **/
+class KotlinGenPlugin implements Plugin<Project> {
+
+  @Override
+  void apply(Project project) {
+    project.afterEvaluate {
+
+      // Grab the release variant
+      // Convenience approach so that we can grab the source sets off of it
+      Collection<LibraryVariant> variants = project.android.libraryVariants
+      LibraryVariant variant = variants.find { v -> v.name == "release" }
+
+      KotlinGenTask genTask = project.task(type: KotlinGenTask, "generateKotlinFor${project.name.capitalize()}") {
```

Comment:
Ah good idea. This actually lets you run all of them at once too if you just run that task from the root project since gradle will run all the matching tasks. Done in 30c4723